packages/jest-repl/package.json

```diff
@@ -10,17 +10,23 @@
   "main": "build/index.js",
   "types": "build/index.d.ts",
   "dependencies": {
+    "@jest/console": "^26.0.1",
+    "@jest/environment": "^26.0.1",
     "@jest/transform": "^26.0.1",
     "@jest/types": "^26.0.1",
+    "@types/yargs": "^15.0.0",
```

Comment:
`@types/yargs` is already present in `devDependencies`, it should be one or the other.

Reply:
Good catch, fixed (and rebased)